### DISCUSSION TITLE
Cannot read property 'writeFile' of null

### DISCUSSION THREAD

**Discussion by Jason-MK** [requarks/wiki#6073] [Help / Questions]
Hello , 

I tried everything possible to fix this issue 

Cannot read property 'writeFile' of null

I'm using Debian 11 , what is the solution I tried to downgrade , I tried upgrade but nothing works , what is the reason for this issue I moved everything 6 months ago to WIKIjs 

I also saw this I tried it , I don't know but it didn't work 

https://github.com/requarks/wiki/discussions/5260

I really need help ?

**Comment by NGPixel** [comment]
Please provide your OS, node version and DB type + version.

**Comment by Jason-MK** [comment]
My OS Debian 10 Buster , postgres version  11.14 , node version v16.14.2

I really hope you can help me ?

best regards

**Comment by Jason-MK** [comment]
@NGPixel  any news ?
